app/apptesting/cosmwasmpool.go

```diff
@@ -22,20 +23,31 @@ const (
 	TransmuterContractName        = "transmuter"
 	TransmuterMigrateContractName = "transmuter_migrate"
 	DefaultCodeId                 = 1
+
+	osmosisRepository         = "osmosis"
+	osmosisRepoTransmuterPath = "x/cosmwasmpool/bytecode"
 )
 
 // PrepareCosmWasmPool sets up a cosmwasm pool with the default parameters.
 func (s *KeeperTestHelper) PrepareCosmWasmPool() cosmwasmpooltypes.CosmWasmExtension {
 	return s.PrepareCustomTransmuterPool(s.TestAccs[0], []string{DefaultTransmuterDenomA, DefaultTransmuterDenomB})
 }
 
-// PrepareCustomTransmuterPool sets up a transmuter pool with the custom parameters.
+// PrepareCustomTransmuterPool sets up a transmuter pool with the default parameters assuming that
+// the transmuter contract is stored under x/cosmwasmpool/bytecode in the Osmosis repository.
 func (s *KeeperTestHelper) PrepareCustomTransmuterPool(owner sdk.AccAddress, denoms []string) cosmwasmpooltypes.CosmWasmExtension {
+	return s.PrepareCustomTransmuterPoolCustomProject(owner, denoms, osmosisRepository, osmosisRepoTransmuterPath)
```

Comment:
Note: these apptesting changes are necessary to allow reusing these helpers from another repository:
https://github.com/osmosis-labs/sqs